themes/default/content/docs/pulumi-cloud/deployments/reference.md

```diff
@@ -228,6 +228,12 @@ Path filters are relative to the repository root, and should reference a file by
 
 As with any other deployment setting, the path filters may be set via the Pulumi Console, using the REST API or defined in code using the Service provider.
 
+### Skip intermediate deployments
+
+When streams of multiple deployments are pushed, they will be executed sequentially until the backlog is completed. In some scenarios, this is not desired as it slows down the execution when they are actually accumulative. By enabling the `Skip intermediate deployments` setting, Pulumi will skip all the consecutive deployments of the same type and will execute only the last one of the serie.
```

Comment:
```suggestion
By default, when multiple deployments are pushed, they will be executed sequentially until the backlog is completed. In some cases, you may wish to only execute the most recent deployment since the changes are accumulative. By enabling the `Skip intermediate deployments` setting, Pulumi will skip all intermediary deployments of the same type and will execute only the latest.
```